bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerHandle.java

```diff
@@ -311,18 +317,24 @@ void writeLedgerConfig(GenericCallback<Void> writeCb) {
     }
 
     /**
-     * Close this ledger synchronously.
-     * @see #asyncClose
+     * {@inheritDoc }
      */
+    @Override
     public void close()
```

Comment:
nit: WriteHandle already has a default implementation of close()

Reply:
We need to keep this overloaded version, because LedgerHandle#close (legacy API) declares to throw a legacy o.a.b.client.BKException, the new signature is with the new o.a.b.client.api.BKException.

If I remove this we are braking source compatibility for legacy applications

Reply:
okay